### DISCUSSION TITLE
How to build this project locally ?

### DISCUSSION THREAD

**Discussion by shresthaujjwal** [openrewrite/rewrite#1305] [Q&A] [answered]
I want to do some contribution but i am not able to build this project from main branch. Does anyone have a instructions on how to build this project ?

**Comment by timtebeek** [comment, marked as answer]
Hi @shresthaujjwal ; Sorry to see you did not get a reaction sooner; discussions were not monitored, but have since been hooked up to our Slack to get notified about any new discssions there.

We have since documented [Building OpenRewrite from source](https://docs.openrewrite.org/reference/building-openrewrite-from-source), which seems a one to one match with your question. Hope that helps, and hope to see you again!
